### PR TITLE
Fix leftover entries after distributed tracker update

### PR DESCRIPTION
**Pull request by jonatanklosko**
Closes https://github.com/phoenixframework/phoenix_pubsub/issues/175.

### CODE REVIEW

File: lib/phoenix/tracker/state.ex [hunk 1]
```diff
@@ -89,7 +89,7 @@ defmodule Phoenix.Tracker.State do
   Atomically updates ETS local entry.
   """
   @spec leave_join(t, pid, topic, key, meta) :: t
-  def leave_join(%State{delta: delta} = state, pid, topic, key, meta) do
+  def leave_join(state, pid, topic, key, meta) do
```
Comment (jonatanklosko) [inline lib/phoenix/tracker/state.ex:92]: When merging leave and join I kept the pattern match, so the join part operated on delta from before leave. Totally my bad, sorry!

**Comment by jonatanklosko** [comment]
@arjan I believe this is it, sorry for the inconvenience and thanks again for the reproduction.

**Comment by arjan** [comment]
Yes looks like this fixes it, thanks!

**Comment by chrismccord** [comment]
❤️❤️❤️🐥🔥

**Comment by chrismccord** [comment]
Released with 2.1.3

**Comment by arjan** [comment]
Amazing, thanks a lot!

**Pull request merged**